packages/cambodia/README.md

```diff
@@ -0,0 +1,53 @@
+# 🇰🇭 Cambodia
+
+[![License: CC BY 4.0](https://img.shields.io/badge/License-CC%20BY%204.0-blue.svg)](https://creativecommons.org/licenses/by/4.0/)
+
+Contains all the provinces of Cambodia:
+* Banteay Meanchey
+* Batdambang
+* Kampong Cham
+* Kampong Chhnang
+* Kampong Spoe
+* Kampong Thum
+* Kampot
+* Kandal
+* Kaoh Kong
+* Kracheh
+* Krong Keb
+* Krong Pailin
+* Krong Preah Sihanouk
+* Mondol Kiri
+* Otdar Meanchey
+* Phnom Penh
+* Pouthisat
+* Preah Vihear
+* Prey Veng
+* Rotana Kiri
+* Siem Reap
+* Steung Treng
+* Svay Rieng
+* Takev
+* Tboung Khmum

```

Comment:
Could you use the [English names](https://en.wikipedia.org/wiki/Provinces_of_Cambodia) please? (e.g. "Battambang" instead of "Batdambang", "Kep" instead "Krong Keb"...)

Or if you know a better source, feel free to give it.

Reply:
I had changed by English names that you provide me... Thanks for advise.